controllers/storageclaim_controller.go

```diff
@@ -274,13 +274,14 @@ func (r *StorageClaimReconciler) reconcilePhases() (reconcile.Result, error) {
 
 		// storageClaimStorageType is the storage type of the StorageClaim
 		var storageClaimStorageType providerclient.StorageType
-		switch r.storageClaim.Spec.Type {
+		claimType := strings.ToLower(r.storageClaim.Spec.Type)
```

Comment:
This was already calculated above as sccType. Please calculate in a single place and reuse

Reply:
done.